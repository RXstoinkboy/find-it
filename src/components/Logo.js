import React from 'react'
import styles from './styles/Logo.module.scss'

export const Logo = () => {
    return (
        <div className={styles.wrapper}>
            <h1 className={styles.logoText}>
                find it
                <div className={styles.logoMark}>!</div>
            </h1>
            <div className={styles.under}></div>
        </div>
    );
};

export default Logo;