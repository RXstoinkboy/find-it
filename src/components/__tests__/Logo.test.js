import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import Logo from '../Logo.js'

test('Logo renders correctly', ()=>{
    const comp = shallow(<Logo />)

    expect(toJson(comp)).toMatchSnapshot()
})