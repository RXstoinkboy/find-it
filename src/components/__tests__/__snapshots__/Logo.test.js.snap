// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<div
  className="wrapper"
>
  <h1
    className="logoText"
  >
    find it
    <div
      className="logoMark"
    >
      !
    </div>
  </h1>
  <div
    className="under"
  />
</div>
`;
